manifests/s/sw4you/Hardcopy/2025.03.31/sw4you.Hardcopy.locale.en-US.yaml

```
@@ -0,0 +1,38 @@
+# Created with komac v2.11.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: sw4you.Hardcopy
+PackageVersion: 2025.03.31
+PackageLocale: en-US
+Publisher: www.hardcopy.de
+PublisherUrl: http://sw4you.de/
```

```suggestion
PublisherUrl: https://sw4you.de/
```

- See https://github.com/microsoft/winget-pkgs/pull/263826#issuecomment-2952771639